test/unit/selector.js

```diff
@@ -608,6 +610,12 @@ QUnit.test( "attributes - special characters", function( assert ) {
 	assert.deepEqual( Sizzle( "input[data-attr='foo_baz\\']']", null, null, attrbad ),
 		q( "attrbad_injection" ),
 		"string containing quote and right bracket" );
+	assert.deepEqual( Sizzle( "input[value=\\30 \\30\\37 ]", null, null, attrbad ),
+		q( "attrbad_leading_digits" ),
+		"identifier containing escaped leading digits with whitespace termination" );
+	assert.deepEqual( Sizzle( "input[value=\\00003007]", null, null, attrbad ),
```

Comment:
Can you explain this assertion?

Reply:
Verifying that hexadecimal escape interpretation stops after six digits, even if there are more.